osi_ippay_payment/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "IPpay Payment",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "summary": "IPpay Payment",
+    "author": "Open Source Integrators",
+    "maintainer": "Open Source Integrators",
+    "website": "http://www.opensourceintegrators.com",
+    "category": "Accounting",
+    "depends": ['payment_authorize'],
```

Comment:
We don't need and don't want to force a dependency on authorize.net
```suggestion
    "depends": ['payment'],
```